src/components/dialog-full-screen/dialog-full-screen.component.js

```diff
@@ -81,6 +81,7 @@ class DialogFullScreen extends Modal {
           titleId="carbon-dialog-title"
           subheader={this.props.subtitle}
           subtitleId="carbon-dialog-subtitle"
+          divider={false}
```

Comment:
Do we still need this then?

Reply:
we have two options:
1. set the `divider={false}` in the component, which will result in not rendering the divider.
2. set the display on the divider to `none`, which will result in hiding the divider.

IMO 1. is a better option